quiz-questions.json

```diff
@@ -1,7 +1,7 @@
 [{   
     "id": 0,
     "question": "Evaluate the follwing expression: let age = 41; console.log(age++): ",
-    "answer": "42", 
+    "answer": "41", 
```

Comment:
This should actually be 41 because since the `++` in  `age++` comes after the age it will evaluate to the current age before adding one to it. If you wanted the answer to be 42 then the question should use `++age`. This way the expression will add one to the age before evaluating since the `++` comes first.

Reply:
hahaha, shows how well I learned from our early lesson.